instrumentation/http-tests/src/main/java/brave/test/http/ITHttp.java

```diff
@@ -157,6 +182,84 @@ protected Tracer tracer() {
     }
   };
 
+  /**
+   * Like {@link #assertSpansReportedKindInOrder(Span.Kind, Span.Kind)} except order isn't enforced.
+   * However, the results will return in the kind order specified.
+   */
+  protected Span[] assertSpansReportedKindInAnyOrder(@Nullable Span.Kind kind1,
+    @Nullable Span.Kind kind2) throws InterruptedException {
+    if (Objects.equals(kind1, kind2)) {
+      throw new AssertionError("Expected test to pass different span kinds");
+    }
+
+    // Intentionally pull both spans first to ensure neither are errors.
+    Span span1 = takeSpan(), span2 = takeSpan();
+
+    Stream.of(span1, span2).forEach(span -> assertThat(span.kind())
```

Comment:
Recommend a normal loop with `Arrays.asList` since it's very similar code but there won't be a lambda in the stack trace if failure

Reply:
great idea